subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/JfrDifferentialFlameGraphGenerator.groovy

```diff
@@ -126,41 +126,15 @@ class JfrDifferentialFlameGraphGenerator implements ProfilerFlameGraphGenerator
     }
 
     enum DetailLevel {
-        RAW(
-            true,
-            true,
-            Arrays.asList("--minwidth", "0.5"),
-            Arrays.asList("--minwidth", "1"),
-            new FlameGraphSanitizer(FlameGraphSanitizer.COLLAPSE_BUILD_SCRIPTS, FlameGraphSanitizer.NORMALIZE_LAMBDA_NAMES)
-        ),
-        SIMPLIFIED(
-            false,
-            false,
-            Arrays.asList("--minwidth", "1"),
-            Arrays.asList("--minwidth", "2"),
-            new FlameGraphSanitizer(FlameGraphSanitizer.COLLAPSE_BUILD_SCRIPTS, FlameGraphSanitizer.NORMALIZE_LAMBDA_NAMES, FlameGraphSanitizer.COLLAPSE_GRADLE_INFRASTRUCTURE, FlameGraphSanitizer.SIMPLE_NAMES)
```

Comment:
Why wouldn't we want the sanitizers for the diffs?

Reply:
We don't use sanitizers for the diffs since the original stacks are already sanitized. We never used sanitizers for the diffs.